doc/quickstart.rst

```diff
@@ -0,0 +1,294 @@
+Quickstart
+==========
+
+Let's learn Ralph by example. Throughout this tutorial, we’ll walk you through:
+
+- adding asset information, licenses and support
+- scanning asset details and configuration
+- access billing information
+
+
+Assets
+^^^^^^
+
+Searching and Filtering of Assets
+---------------------------------
+
+Let's start with the main screen. Here you find all your hardware assets which
+are in the database. Use the left column for filtering of the assets.
+
+There are two types of assets -- devices and parts. A device could be a blade
+server, and a part is a component of this server, for example memory or hard
+disk drive. A part can be assigned to a single device at a time. You can move
+parts from one device to another when you need it.
+
+
+Adding Assets
+-------------
+
+If you want add data center asset you need to fill a few required fields.
+Look at the screen below (you can zoom-in the image in your browser if you
+want) The fields in bold are required. Rest of fields are optional.
+
+.. image:: _static/add-data-center-asset-form.png
+    :scale: 30%
+
+Let's describe each of the fields:
+
+- **Basic info**:
+    - **Type** - a read only field for data center (example: )
+    - **Model** - choose a model for asset, (you can type a couple of letters
+      to search for a given model. If no result is found, just click "Add"
+      button to add it) (example: )
+    - **Inventory number** - an alphanumeric value in your stock (example: )
+    - **Warehouse** - the place where the asset is located (example: )
+    - **Location** - a more exact location of the device in the building/room (example: )
+    - **Status** - an asset's lifetime indicator. Newly bought assets has
+      status "new". You can change it as required according to your own work
+      flow (example: )
+    - **Task url** - url to task in ticket system (example: )
+    - **Additional remarks** - field for various data (example: )
+    - **Service name** - service name to which this asset belongs (example: )
+    - **Property of** - company to which this asset belongs (example: )
+    - **Hostname** - read-only field telling the name of host (from Ralph-core
+      device) (example: )
+    - **Service catalog** - points to ``service catalog`` which asset belongs
```

Comment:
*service from the ``service catalog``*? as it doesn't point to the whole catalogue ;)